src/main/java/co/omise/models/SourceType.java

```diff
@@ -1,26 +1,46 @@
 package co.omise.models;
 
+import com.fasterxml.jackson.annotation.JsonEnumDefaultValue;
 import com.fasterxml.jackson.annotation.JsonProperty;
 
 import java.lang.reflect.Field;
 
 public enum SourceType {
-    @JsonProperty("alipay")Alipay,
-    @JsonProperty("barcode_alipay")BarcodeAlipay,
-    @JsonProperty("bill_payment_tesco_lotus")BillPaymentTescoLotus,
-    @JsonProperty("econtext")Econtext,
-    @JsonProperty("installment_bay")InstallmentBay,
-    @JsonProperty("installment_bbl")InstallmentBbl,
-    @JsonProperty("installment_first_choice")InstallmentFirstChoice,
-    @JsonProperty("installment_kbank")InstallmentKbank,
-    @JsonProperty("installment_ktc")InstallmentKtc,
-    @JsonProperty("internet_banking_bay")InternetBankingBay,
-    @JsonProperty("internet_banking_bbl")InternetBankingBbl,
-    @JsonProperty("internet_banking_ktb")InternetBankingKtb,
-    @JsonProperty("internet_banking_scb")InternetBankingScb,
-    @JsonProperty("paynow")Paynow,
-    @JsonProperty("points")Points,
-    @JsonProperty("truemoney")TrueMoney;
+    @JsonEnumDefaultValue
+    @JsonProperty("unknown")
```

Comment:
Still thinking about is there is away to keep the raw value with `Unknown` type. However, I'm OK to go with this way then we find the way out later. 🙂